perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -438,6 +490,37 @@ def BuildBenchmarkScriptCommand(args):
                                          LATENCY_UNIT,
                                          metadata)
 
+    if (FLAGS.object_storage_scenario == 'all' or
+        FLAGS.object_storage_scenario == 'api_multistream'):
+
+      write_start_time = (time.time() +
+                          MULTISTREAM_FIXED_DELAY +
+                          MULTISTREAM_DELAY_PER_STREAM * FLAGS.num_streams)
+
```

Comment:
Add some info logs here to indicate the progress of this test:

For example, add progress information at each step start/stop time.
log.info("starting write multi-stream test")

log.info("write mult-stream test completed successfully, starting read multi-stream test...)

etc.